src/cr/cube/stripe/assembler.py

```diff
@@ -367,6 +368,49 @@ def _display_order(self):
         return CollatorCls.display_order(self._rows_dimension, self._empty_row_idxs)
 
 
+class _SortByLabelHelper(_BaseOrderHelper):
+    """Orders rows by the value of a specified measure."""
+
+    @lazyproperty
+    def _display_order(self):
+        """tuple of signed int idx for each row of stripe.
+
+        Negative values represent inserted-vector locations. Returned sequence reflects
+        insertion, hiding, pruning, and ordering transforms specified in the
+        rows-dimension.
+        """
+        # --- fall back to payload order if sort-by-value failes
+        try:
+            return SortByValueCollator.display_order(
+                self._rows_dimension,
+                self._element_values,
+                self._subtotal_values,
+                self._empty_row_idxs,
+            )
+        except ValueError:
+            return PayloadOrderCollator.display_order(
+                self._rows_dimension, self._empty_row_idxs
+            )
```

Comment:
Same for this. The implementation is identical so I'm thinking we should extract a sort-by-value base class.